data/shaders/atmosphere.vert

```diff
@@ -47,22 +47,14 @@ void main()
 	highp vec3 unprojectedVertex = skyColor.xyz;
 	highp float Y = skyColor.w;
 	highp float x,y;
-	if (Y>0.01)
```

Comment:
Deleting this distinction disables the separation between day and night vision. I have to go into more details here and dig out the original papers, but to my understanding our current code is close to the published models, and the colors look like most other implementations on various game engines.

Reply:
No, this just removes the redundancy with respect to `xyYToRGB.glsl`, where this same check is already done, and where x,y values are overridden.